Core/include/Acts/Material/VolumeMaterialMapper.hpp

```diff
@@ -60,6 +61,13 @@ class VolumeMaterialMapper {
  public:
   using StraightLinePropagator = Propagator<StraightLineStepper, Navigator>;
 
+  /// @brief selector for finding surface
+  struct BoundSurface {
+    bool operator()(const Surface& sf) const {
+      return (sf.geometryId().boundary() != 0);
+    }
+  };
+
```

Comment:
Can you give this a more descriptive name, e.g. `BoundarySurfaceSelector` or something along those lines. Does this have to be a public type?

Reply:
No it does have to be public, I will switch it private with the material volume one too.